src/scan/camera/Camera.tsx

```diff
@@ -81,20 +81,22 @@ const Camera = ({ navigation }: TProps) => {
     return <CameraSettings />;
   }
 
-  if (!hasCameraPermission || device == null) {
+  if (device == null) {
     return <Spinner />;
   }
 
   return (
     <View onLayout={handleLayoutChange} style={StyleSheet.absoluteFill}>
       <FocusAwareStatusBar backgroundColor={theme.palette.neutral['900']} barStyle="light-content" translucent />
-      <VisionCamera
-        device={device}
-        frameProcessor={frameProcessor}
-        frameProcessorFps={5}
-        isActive={isActive}
-        style={{ height: overlayDimensions[1], width: overlayDimensions[0] }}
-      />
+      {overlayDimensions[0] !== 0 &&
```

Comment:
kan overlayDimensions default dan niet undefined zijn? Precies raar om te checken op 0

Reply:
Heeft niets met uw pr te maken, maar zou het duidelijker zijn om hier van een object te maken met height en width? Of kijk ik iets over het hoofd

Reply:
Die eerste: das een inititialze op 0, en die gaat ook door een andere hook waar ook een number verwacht wordt, dus die durf ik niet zo goed op undefined te zetten. 

Die 2de: done